rom/dev/tests/test_image_validation.rs

```diff
@@ -1339,6 +1339,12 @@ fn cert_test_with_custom_dates() {
     hw.upload_firmware(&image_bundle.to_bytes().unwrap())
         .unwrap();
 
+    hw.step_until_output_contains("[exit] Launching FMC")
+        .unwrap();
+
+    let result = hw.mailbox_execute(0x1000_0001, &[]);
```

Comment:
Can we use CONSTANTS for these command IDs?

Reply:
Sounds good; will add those.